### DISCUSSION TITLE
CK?

### DISCUSSION THREAD

**Discussion by JoeMheh** [3kh0/ext-remover#528] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Idk if anyone has talked about this but my school uses CK and another idk but I think my school hates me since the CK keeps blocking everything like this one time I was going to look up "7 squared" since I'm lazy and don't feel like doing the work and I get blocked to search that like it makes no sense. So my question is will the extension remover ever work for CK? If it will never work someone just tell me, but if there is a chance that it is possible tell me also cause like I really want to know.
Also Killcurly won't work for CK it would break the extensions but CK just loads and loads if broken with killcurly

**Comment by funnypenguin100** [comment, marked as answer]
I have no idea what CK is- but if it is an extension, it should work... (unless your school blocked that)

**Reply by JoeMheh** [reply]
CK-Authenticator and CK-Express TP is what my school uses idk which is blocking stuff tho

**Reply by SomePVZPlayerMatt** [reply]
my district has it too (the former)

**Reply by Ambush4567** [reply]
my school uses CK-Express TP to block stuff and none of the exploits work for me (i'm not risking SH1mmer), could you create a point blank or other exploit to get over it?
